SpringProject/src/it/java/com/gradproject2019/user/UserControllerIT.java

```diff
@@ -141,6 +144,28 @@ public void shouldReturn401WhenUnauthorisedEdit() throws URISyntaxException {
         Assert.assertEquals("User unauthorized to perform action.", response.getBody().getMessage());
     }
 
+    @Test
+    public void shouldReturn200AndEmptyListWhenDatabaseEmpty() throws URISyntaxException {
+        URI uri = new URI(baseUri);
+        clearRepositories();
+
+        ResponseEntity<List<UserResponseDto>> response = getUserList(uri);
+
+        Assert.assertEquals(200, response.getStatusCodeValue());
+        Assert.assertEquals(true, response.getBody().isEmpty());
+    }
+
+    @Test
+    public void shouldReturn200AndListOfUsersWhenDatabasePopulated() throws URISyntaxException {
```

Comment:
Again the test name here could be changed to `WhenUserExists`

Seeing as you're only adding 1 user, it would be nice to assert the size is 1 here as well